storybook/src/components/Header/Header.stories.tsx

```diff
@@ -3,14 +3,27 @@
  * Copyright Gemeente Amsterdam
  */
 
-import { PageMenu } from '@amsterdam/design-system-react'
+import { PageMenu, Screen } from '@amsterdam/design-system-react'
 import { Header } from '@amsterdam/design-system-react/src'
-import { SearchIcon } from '@amsterdam/design-system-react-icons'
+import { MenuIcon, SearchIcon } from '@amsterdam/design-system-react-icons'
 import { Meta, StoryObj } from '@storybook/react'
 
+const MenuButton = (
+  <PageMenu.Button icon={MenuIcon} onClick={() => {}}>
+    Menu
+  </PageMenu.Button>
+)
+
 const meta = {
   title: 'Components/Containers/Header',
   component: Header,
+  decorators: [
+    (Story) => (
+      <Screen>
+        <Story />
+      </Screen>
+    ),
+  ],
```

Comment:
To prevent the body from having padding, you can't use viewport settings now to see phone screens for example.

```suggestion
  decorators: [
    (Story) => (
      <Screen>
        <Story />
      </Screen>
    ),
  ],
  parameters: {
    layout: 'fullscreen',
  },
```

Reply:
The ‘English’ and ‘Mijn Amsterdam’ items not disappearing was due to a different bug. I’d rather not use fullscreen layout here because it removes the padding that the Header will have in real life (even though its sizes will be different) and because this `Header` is not the entire header – it’s just a container that still needs a parent grid for the correct layout. The media query may kick in 20 pixels late or early, but that’s acceptable.